spec/support/matchers/accessibility.rb

```diff
@@ -289,10 +310,14 @@ def fieldset_legend_name(element)
 end
 
 def expect_page_to_have_no_accessibility_violations(page, validate_markup: true)
-  expect(page).to be_axe_clean.according_to :section508, :"best-practice", :wcag21aa
+  expect(page).to be_axe_clean.according_to(
+    :section508, :"best-practice",
+    :wcag21aa
+  ).skipping('aria-allowed-role')
```

Comment:
Not sure if skipping all role attributes is a good solution here.

Reply:
Can I modify `tag_decorative_svgs_with_role` to compensate, I wonder.

Reply:
I had a feeling that if we need to bypass typical best practices for `img` `role` that we might have to do something like this, so it's not totally unexpected to me. I'd wonder if we could at least limit it to the very specific case we're trying to exempt, i.e. `img`  `role` and not [anything else that Axe might be looking for](https://dequeuniversity.com/rules/axe/4.7/aria-allowed-role.

Reply:
I haven't done any research to validate the idea, but I'd also wonder if we could mark the image as `aria-hidden="true"` if it's decorative anyways. My understanding is that the expected behavior for a screen reader with a decorative image (`alt=""`) is to skip it entirely and omit it from the page structure, which is the same as what happens with `aria-hidden="true"`.

Reply:
Yeah, Zach's selector up above for the decorative svg function worked great there. 🙂